src/components/media_manager/CMakeLists.txt

```diff
@@ -99,22 +46,60 @@ include_directories (
     ${JSONCPP_INCLUDE_DIRECTORY}
     ${CMAKE_BINARY_DIR}/src/components/
     ${COMPONENTS_DIR}/policy/include/
-    ${default_includes}
     ${LOG4CXX_INCLUDE_DIRECTORY}
 )
 
-set (SOURCES
-    ${COMPONENTS_DIR}/media_manager/src/media_adapter_impl.cc
-    ${COMPONENTS_DIR}/media_manager/src/audio/from_mic_recorder_listener.cc
-    ${COMPONENTS_DIR}/media_manager/src/audio/audio_stream_sender_thread.cc
-    ${COMPONENTS_DIR}/media_manager/src/streamer_listener.cc
-    ${COMPONENTS_DIR}/media_manager/src/media_manager_impl.cc
+set(PATHS
+  ${CMAKE_CURRENT_SOURCE_DIR}/include
+  ${CMAKE_CURRENT_SOURCE_DIR}/src
 )
 
-add_library("MediaManager" ${SOURCES} ${default_sources})
+set(EXCLUDE_PATHS
+  video_stream_to_file_adapter.cc
+)
+
+set(LIBRARIES
+    ProtocolLibrary
+)
+
+if(EXTENDED_MEDIA_MODE)
+  find_package(Gstreamer-1.0 REQUIRED)
+  find_package(Glib-2.0 REQUIRED)
+  find_package(PkgConfig)
+  pkg_check_modules(GLIB2 REQUIRED glib-2.0)
+  add_definitions(${GLIB2_CFLAGS})
+
+  include_directories(
+    ${GSTREAMER_gst_INCLUDE_DIR}
+    ${GSTREAMER_gstconfig_INCLUDE_DIR}
+    ${GLIB_glib_2_INCLUDE_DIR}
+  )
+  list(APPEND EXCLUDE_PATHS
+
+  )
+  list(APPEND LIBRARIES
+    ${GSTREAMER_gstreamer_LIBRARY}
+    pulse-simple
+    pulse
+    gobject-2.0
+    glib-2.0
+  )
+else()
+  list(APPEND EXCLUDE_PATHS
+	a2dp_source_player_adapter.cc
```

Comment:
@AByzhynar I am completely agree with you that having some consistency in style is essential and applying style checker make perfectly sense. Until then enforcing the consistency is just a personal responsibility for each contributor.